lib/spack/spack/cmd/unload.py

```diff
@@ -4,20 +4,59 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import argparse
-from spack.cmd.common import print_module_placeholder_help
+import sys
 
-description = "remove package from environment using `module unload`"
-section = "modules"
+import llnl.util.tty as tty
+
+import spack.cmd
+import spack.environment as ev
+import spack.util.environment
+import spack.user_environment as uenv
+
+description = "remove package from the user environment variables`"
+section = "user environment"
 level = "short"
 
 
 def setup_parser(subparser):
```

Comment:
This should probably have a `--all` option to unload every loaded hash.

Reply:
Now it does.